MM2RandoLib/Extensions/StringExtensions.cs

```diff
@@ -0,0 +1,122 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace MM2Randomizer.Extensions
+{
+    public static class StringExtensions
+    {
+        //
+        // Public Methods
+        //
+
+        public static Byte[] AsIntroString(this String in_String)
+        {
+            if (null != in_String)
+            {
+                Byte[] convertedString = new Byte[in_String.Length];
+                Int32 index = 0;
+
+                foreach (Char c in in_String)
+                {
+                    if (true == StringExtensions.IntroCharacterLookup.TryGetValue(c, out Byte convertedCharacter))
+                    {
+                        convertedString[index] = convertedCharacter;
+                    }
+                    else
+                    {
+                        convertedString[index] = IntroCharacterLookup['?'];
+                    }
+
+                    index++;
+                }
+
+                return convertedString;
+            }
+            else
+            {
+                return null;
+            }
+        }
+
+
+        //
+        // Private Constants
+        //
+
+        private static readonly Dictionary<Char, Byte> IntroCharacterLookup = new Dictionary<Char, Byte>()
+        {
+            { ' ', 0x00 },
+            { '0', 0xA0 },
+            { '8', 0xA1 },
+            { '2', 0xA2 },
+            { '©', 0xA3 },
+            { '™', 0xA4 }, // tm
+            { '9', 0xA5 },
+            { '7', 0xA6 },
+            { '1', 0xA7 },
+            { '3', 0xA8 }, // check
+            { '4', 0xA9 }, // check
+            { '5', 0xAA }, // check
+            { '6', 0xAB }, // check
+            { '|', 0xC0 }, // Blank space, change later? also, there may be a blank row (B) to use...
```

Comment:
Is it later? Should we change this now?

Reply:
I'm just removing the comment and putting a better (?) one.